helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -36,6 +36,12 @@ data:
         name: '{{ .Values.Installation.V1.Name }}'
         {{- if .Values.Installation.V1.Guest }}
         guest:
+          ipam:
+            network:
+              CIDR: '{{ .Values.Installation.V1.Guest.IPAM.NetworkCIDR }}'
+              subnetMaskBits: '{{ .Values.Installation.V1.Guest.IPAM.CIDRMask }}'
+              privateSubnetMaskBits: '{{ .Values.Installation.V1.Guest.IPAM.PrivateSubnetMask }}'
+              publicSubnetMaskBits: '{{ .Values.Installation.V1.Guest.IPAM.PublicSubnetMask }}'
```

Comment:
We need to talk about this here. I am not sure about the implications and the state. Are the installations prepared for this? Where do the values come from? Are these things optional?

Reply:
`installations` are prepared. These are the configs currently used by `kubernetesd`. There are also defaults for these values in code IIRC.

Reply:
`installations` are prepared. These are the configs currently used by `kubernetesd`. There are also defaults for these values in code IIRC.

Reply:
I've heard we can maybe hide public/private computation inside aws-oeprator?

Also can we do it like that: `CIDR: '{{ .Values.Installation.V1.Guest.IPAM.NetworkCIDR }}/{{ .Values.Installation.V1.Guest.IPAM.CIDRMask }}'` ? This will be easier to merge those values in installations later on. They shouldn't be separate. What we have currently isn't a valid CIDR notation.